### PR TITLE
change notebook to follow qutip PR#2170

### PR DESCRIPTION
**Pull request by tamakoshi2001**
I made changes on arguments of functions in visualization.py in [#2170](https://github.com/qutip/qutip/pull/2170). This PR follows it.

### CODE REVIEW

File: tutorials-v5/visualization/visualization-exposition.md [hunk 1]
```diff
@@ -72,8 +73,8 @@ qt.matrix_histogram_complex(rho.full());
 ```python
 H0 = qt.tensor(sigmaz(), identity(2)) + qt.tensor(identity(2), sigmaz())
 Hint = 0.1 * qt.tensor(sigmax(), sigmax())
-
-qt.plot_energy_levels([H0, Hint], figsize=(8, 4));
+fig = plt.figure(figsize=(8,4))
+qt.plot_energy_levels([H0, Hint], fig=fig);
```
Comment (nwlambert) [inline tutorials-v5/visualization/visualization-exposition.md:77]: could demonstrate using h_labels here?

Comment (tamakoshi2001) [reply tutorials-v5/visualization/visualization-exposition.md]: Hi. I added a h_labels demo.
h_label_demo

**Comment by Ericgig** [comment]
@tamakoshi2001 
Could you fix the PR, here pep8 issues makes the tests fails.

**Comment by tamakoshi2001** [comment]
@Ericgig I fixed it. Could you run the test again?

**Comment by Ericgig** [comment]
Test failed again. I think it's not up to date with the latest changes done in 2170.

**Comment by tamakoshi2001** [comment]
@Ericgig Hi. I fixed the issues. I also found and fixed some hidden issues.

**Review by Ericgig — APPROVED** [review]

**Pull request merged**